resources/multisite_drupal_standard.make

```diff
@@ -1016,9 +1016,9 @@ projects[ec_resp][download][tag] = 2.3.4
 projects[atomium][type] = theme
 projects[atomium][download][type] = git
 projects[atomium][download][url] = https://github.com/ec-europa/atomium.git
-projects[atomium][download][branch] = 7.x-2.x
```

Comment:
According with the documentation - https://webgate.ec.europa.eu/fpfis/wikis/display/MULTISITE/Contributed+modules#version, it's not necessary to change this version.

Reply:
HI @voidtek, look deeper in the example you are missing the commit hash.